src/app/(proper_react)/redesign/(authenticated)/user/dashboard/fix/high-risk-data-breaches/FraudAlertModal.tsx

```diff
@@ -0,0 +1,139 @@
+
+  return (
+    <>
+      <button
+        {...buttonProps}
+        ref={triggerRef}
+        className={styles.triggerButton}
+      >
+        <QuestionMarkCircle
+          alt={l10n.getString(
+            "fix-flow-data-broker-profiles-view-data-broker-profiles-more-dialog-trigger-label"
+          )}
+          width={18}
+          height={18}
+        />
+      </button>
+      {overlayTriggerState.isOpen && (
+        <ModalOverlay
+          state={overlayTriggerState}
+          {...overlayProps}
+          isDismissable={true}
+        >
+          <Dialog
+            title={l10n.getString("ssn-modal-title")}
+            illustration={<Image src={FraudAlertDialogIllustration} alt="" />}
+          >
+            <div className={styles.dialogContents}>
+              <p>
+                {l10n.getFragment("ssn-modal-description-fraud-part-one", {
+                  elems: {
+                    b: <strong />,
+                  },
+                })}
+                &nbsp;
+                {l10n.getString("ssn-modal-description-fraud-part-two")}
+              </p>
+              <p>
+                {l10n.getFragment(
+                  "ssn-modal-description-freeze-credit-part-one",
+                  {
+                    elems: {
+                      b: <strong />,
+                    },
+                  }
+                )}
+                &nbsp;
+                {l10n.getFragment(
+                  "ssn-modal-description-freeze-credit-part-two",
+                  {
+                    elems: {
+                      link_to_equifax_info: (
+                        <a
+                          href={equifaxLink}
+                          rel="noopener noreferrer"
+                          target="_blank"
+                        />
+                      ),
+                      link_to_experian_info: (
+                        <a
+                          href={experianLink}
+                          rel="noopener noreferrer"
+                          target="_blank"
+                        />
+                      ),
+                      link_to_transunion_info: (
+                        <a
+                          href={transunionLink}
+                          rel="noopener noreferrer"
+                          target="_blank"
+                        />
+                      ),
+                    },
+                  }
+                )}
+              </p>
+              <p>
+                <Link
+                  href={fraudAlertLink}
+                  rel="noopener noreferrer"
+                  target="_blank"
+                >
+                  {l10n.getString("ssn-modal-learn-more")}
+                </Link>
+              </p>
+              <div className={styles.confirmButtonWrapper}>
+                <Button
+                  variant="primary"
+                  onClick={() => overlayTriggerState.close()}
```

Comment:
Currently I can't activate the button by pressing <kbd>Enter</kbd>, which I believe should be handled by `onPress`.

```suggestion
                  onPress={() => overlayTriggerState.close()}
```